### PR TITLE
fix: mock docs

### PR DESCRIPTION
**Pull request by zllkjc**
## Types of changes

- [ ] Docs change / Dependency upgrade
- [ ] Bug fix
- [ ] New feature / Improvement
- [ ] Refactoring
- [ ] Breaking change

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx [hunk 1]
```diff
@@ -286,9 +286,9 @@ Modern.js 支持了 React 18 的流式渲染，可以通过如下配置启用：
 {
   "server": {
     "ssr": {
-      "mode": "stream"
-    }
-  }
+      "mode": "stream",
```
Comment (chenjiahan) [inline packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx:289]: 这好像是个 json block，没法加逗号

File: packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx [hunk 2]
```diff
@@ -413,8 +413,8 @@ export default Page;
 
 也可以通过 `useAsyncValue` 获取 Data Loader 返回的异步数据。例如：
 
-```
-```ts title='page.tsx'
+
+```tsx title='page.tsx'
```
Comment (chenjiahan) [inline packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx:417]: how about english document

File: packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx (hunk 1 above)
Comment (zllkjc) [reply packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx:293]: 改成实际的例子了

File: packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx (hunk 2 above)
Comment (zllkjc) [reply packages/toolkit/main-doc/zh/guides/advanced-features/ssr.mdx:421]: done 啦

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**